### PR TITLE
ExplorePrizePoolsHeader: Styled Dismissable Info + Buttons

### PR DESCRIPTION
**Pull request by Ncookiez**
- Styled explainer section for desktop and mobile.
- The section can now be dismissed and brought back up through buttons.

By default, the explanation will be displayed with a 'Dismiss' button in its bottom-right. If clicked, the explanation will be hidden and instead a "How does this work?" button will be displayed instead. Was debating the position of this button, but it looks decent left-aligned as of now imo.

### Desktop:

image

image

### Mobile:

image

image

### CODE REVIEW

File: src/components/ModalViews/ExplorePrizePoolsView/ExplorePrizePoolsHeader.tsx [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
+
+  if (dismissed) {
+    return (
+      <Button
+        onClick={enable}
+        className='mb-6 opacity-75 hover:opacity-100'
+        size={ButtonSize.sm}
+        theme={ButtonTheme.transparent}
+      >
+        How does this work?
```
Comment (dylandesrosier) [inline src/components/ModalViews/ExplorePrizePoolsView/ExplorePrizePoolsHeader.tsx:18]: Localize!

File: src/components/ModalViews/ExplorePrizePoolsView/ExplorePrizePoolsHeader.tsx [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
+          {t('learnMore')}
+        </ExternalLink>
+      </div>
+      <Button
+        onClick={dismiss}
+        className='mt-auto ml-auto flex-shrink-0 opacity-75 hover:opacity-100'
+        size={ButtonSize.sm}
+        theme={ButtonTheme.transparent}
+      >
+        Dismiss
```
Comment (dylandesrosier) [inline src/components/ModalViews/ExplorePrizePoolsView/ExplorePrizePoolsHeader.tsx:37]: Localize!

**Review by dylandesrosier — CHANGES_REQUESTED** [review]

File: src/components/ModalViews/ExplorePrizePoolsView/ExplorePrizePoolsHeader.tsx (hunk 1 above)
Comment (Ncookiez) [reply src/components/ModalViews/ExplorePrizePoolsView/ExplorePrizePoolsHeader.tsx]: Fixed through ed8e1f1!

File: src/components/ModalViews/ExplorePrizePoolsView/ExplorePrizePoolsHeader.tsx (hunk 2 above)
Comment (Ncookiez) [reply src/components/ModalViews/ExplorePrizePoolsView/ExplorePrizePoolsHeader.tsx]: Fixed through ed8e1f1!

**Review by dylandesrosier — APPROVED** [review]

**Pull request merged**